superset-frontend/package.json

```diff
@@ -89,7 +89,7 @@
     "@superset-ui/legacy-plugin-chart-treemap": "^0.17.85",
     "@superset-ui/legacy-plugin-chart-world-map": "^0.17.85",
     "@superset-ui/legacy-preset-chart-big-number": "^0.17.86",
-    "@superset-ui/legacy-preset-chart-deckgl": "^0.4.11",
+    "@superset-ui/legacy-preset-chart-deckgl": "^0.4.12",
```

Comment:
Is this related to the PR at hand?

Reply:
yep. Had to bump deck.gl to a new version that allows the new superset-ui version as a peer.